web/core/src/transformers.ts

```diff
@@ -6,15 +6,14 @@ import {
 import { GameStep, PokerGameStep } from "./types";
 
 export const processEpisodeData = (
-  steps: any[],
```

Comment:
I ~think this also needs the corresponding change in kaggleazure, right?

Reply:
@develra yes, have that teed up next